package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "minesweeper",
+  "version": "1.0.0",
+  "description": "Minesweeper game written in TypeScript",
+  "main": "src/index.js",
```

Comment:
I think we can remove this

Reply:
Remove the whole thing?  These are pretty standard properties in package.json.

Reply:
I'm talking about just the `main` property. The `main` property is so if someone installs this package (which we aren't going to publish) they can "run it" using this script right? Except that doesn't make sense, this file wont exist and we aren't going to publish this as a package, so I think the line is just bloat.